python/neutron-understack/neutron_understack/nautobot.py

```diff
@@ -38,7 +38,7 @@ def _log_and_raise_for_status(self, response):
         try:
             response.raise_for_status()
         except HTTPError as error:
-            LOG.error("Nautobot error: %(error)s", {"error": error})
+            LOG.error("Nautobot error: %s %s", error, response.content)
             raise NautobotError() from error
```

Comment:
Honestly some kind of detailed exception is what should be getting thrown here rather than the default which has a message body still of "Nautobot error"